### DISCUSSION TITLE
Filtro in en consulta search_read

### DISCUSSION THREAD

**Discussion by amoradel** [tmijail/Odoo-Power-BI-Connector#18] [Q&A] [answered]
Hola,

Es posible realizar un filtro in, parecido a `['sale_id', 'in', sale_ids]` dentro de los filtros del search_query tomando una columna de la tabla sale_order?

![image](https://github.com/tmijail/Odoo-Power-BI-Connector/assets/10636860/c2a223cc-c9bc-4854-9968-370063b8cae9)

**Comment by tmijail** [comment, marked as answer]
Sí. Cambia `sale_order.id` por `sale_order[id]`

**Reply by amoradel** [reply]
Te lo agradezco mucho Mijail,

¡Sos un crack!
